packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -116,6 +118,10 @@ export class BlockList extends Component {
 					scrollViewStyle={ { flex: isRootList ? 1 : 0 } }
 					data={ blockClientIds }
 					keyExtractor={ identity }
+					extraData={
+						shouldShowInsertionPointBefore ||
```

Comment:
we could define a `forceRefresh` variable that has this value and use it here, it would make this change more explicit

Reply:
This would increase code readability